web/routes.go

```diff
@@ -77,6 +77,10 @@ func (sc *ServiceConfig) getRoutes() []rest.Route {
 		rest.Route{"PUT", "/services/:serviceId/startService", gz(sc.authorizedClient(restStartService))},
 		rest.Route{"PUT", "/services/:serviceId/stopService", gz(sc.authorizedClient(restStopService))},
 
+		// Services (migration)
+		rest.Route{"GET", "/servicemigration/:serviceId", sc.authorizedClient(restGetServicesForMigration)},
```

Comment:
After some more thought, I don't think it makes sense to create a new endpoint just for this.
I'm paraphrasing slightly, but 
`GET /servicemigration/:serviceId` should be replaced with `GET /services/:servicedId?includeChildren=true`.
The point is that returning a list of a service plus it's children is generic operation for services in general, so we should be able to incorporate that function into the existing `/services` namespace.